From: Emmanuel Arias <eamanu@yaerobi.com>
Date: Wed, 4 Nov 2020 16:31:06 -0300
Subject: Remove cmake extension

Remove cmake extension because is no package on Debian yet. This patch
must be removed after sphinxcontrib.moderncmakedomain is packaged on Debian.

Last-Update: 2022-03-27
---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: scikit-build/docs/conf.py
===================================================================
--- scikit-build.orig/docs/conf.py	2023-07-15 13:42:22.400662055 -0300
+++ scikit-build/docs/conf.py	2023-07-15 13:42:22.396662001 -0300
@@ -40,7 +40,7 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
-    'sphinxcontrib.moderncmakedomain',
+#    'sphinxcontrib.moderncmakedomain',
     'sphinx.ext.autodoc',
     'sphinx.ext.viewcode',
     'sphinx_issues',
